Docker/doc.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04
```

Comment:
Workflow uses `ubuntu-latest`, here we have 20.04. Just wanted to check if this was intentional.

Reply:
Here ubuntu 20.04 is not necessary. We can just use ubuntu-latest. I just copy the dockerfile for building ParallelZone, in which ubuntu 20.04 is necessary to get the correct version of gcc. Thanks for catching this! I will update the file.

Reply:
Updated to ubuntu-latest.